doc/cephfs/client-config-ref.rst

```diff
@@ -1,6 +1,48 @@
-========================
- Client Config Reference
-========================
+Client Configuration
+====================
+
+Updating Client Configuration
+-----------------------------
+
+Certain client configurations can be applied at runtime. To check if a configuration option can be applied (taken into affect by a client) at runtime, use the `config help` command:
+
+::
+
+   ceph config help <option>
+
+   e.g.,
```

Comment:
It would be nice if we only have commands inside literal block.

Reply:
done